vNext/src/OrleansRuntime/error.txt

```diff
@@ -0,0 +1,11 @@
+The process cannot access the file 'D:\microsoft\orleans-xiazen\vNext\src\OrleansRuntime\obj\Debug\netstandard1.5\Generated\OrleansRuntime.codegen.cs' because it is being used by another process.
```

Comment:
delete :) We merged a fix so that these are no longer generated, so it wont be a problem any longer